### ISSUE TITLE
question about the sigma2 prior loss increasing during the training 

### ISSUE THREAD

**Issue by maxandchen** [google/uis-rnn#60]
## Describe the question
A clear and concise description of what the question is.
a TDNN model is trained to extract embedding called  x-vector ,so i use  x-vector instead of d-vector . During the uis-rnn  training ,my sigma2 prior loss keep increasing  although the traing loss is decreasing , I wonder if this is abnormal ?
![image](https://user-images.githubusercontent.com/55789492/65654062-153c8400-e04a-11e9-988d-47d1f9eadabf.png)
## My background

Have I read the `README.md` file?
* yes/no - if you answered no, please stop filing the issue, and read it first

Have I searched for similar questions from closed issues?
* yes/no - if you answered no, please do it first

Have I tried to find the answers in the paper *Fully Supervised Speaker Diarization*?
* yes/no

Have I tried to find the answers in the reference *Speaker Diarization with LSTM*?
* yes/no

Have I tried to find the answers in the reference *Generalized End-to-End Loss for Speaker Verification*?
* yes/no

**Comment by wq2012** [comment]
@maxandchen d-vector and x-vector are basically the same thing, just being given different names by Google and JHU respectively, using slightly different network topology, data augmentation, and loss functions. Shouldn't make much difference.

I think the increasing sigma2 prior loss is very normal. It's just a regularization term. Sacrificing regularization constraints to better fit data is quite common.

**Comment by wq2012** [comment]
@maxandchen BTW does your x-vector model use LSTM/GRU or purely feed-forward network? Feed-forward network usually has much much worse performance than LSTM/GRU in speaker recognition.
